### PR TITLE
remove cluster logging e2e test from flaky suite

### PR DESCRIPTION
**Pull request by freehan**
ref:  #17873

### CODE REVIEW

**Comment by freehan** [comment]
@k8s-bot unit test this

**Comment by freehan** [comment]
@k8s-bot unit test this

**Comment by saad-ali** [comment]
Looks like there was 1 failure for this test since the fix (#18783) was merged:
* http://kubekins.dls.corp.google.com/view/Flaky/job/kubernetes-e2e-gce-flaky/9105/
```
22:17:56   Dec 20 22:17:53.872: Got incorrect count for node 2 index 0: 2
```
Stability: <1% failure rate

LGTM

File: test/e2e/es_cluster_logging.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ import (
 )
 
 // Flaky issue #17873
```
Comment (saad-ali) [inline test/e2e/es_cluster_logging.go]: Remove comment.

**Comment by freehan** [comment]
@k8s-bot unit test this

**Comment by spxtr** [comment]
http://kubekins.dls.corp.google.com/job/kubernetes-e2e-gce-parallel-flaky/12589/
http://kubekins.dls.corp.google.com/job/kubernetes-e2e-gce-flaky/9451/

This is still failing occasionally in the flaky suites.

**Comment by freehan** [comment]
@spxtr Yes. I saw that. It starts flaky again right after this PR got submitted.

**Comment by goltermann** [comment]
@freehan can you push this one in to resolve #17873?

**Comment by goltermann** [comment]
nm, just realized it was in the queue

**Comment by spxtr** [comment]
This is still failing ~1% of the time in `kubernetes-e2e-gce-flaky`, but it looks good in `gce-parallel-flaky`.

**Pull request merged**